sdk/react/panels/AudioAndCCSelectionPanel.js

```diff
@@ -58,15 +58,8 @@ class AudioAndCCSelectionPanel extends Component {
   }
 
   onAudioTrackSelected = (name) => {
-    const localizedTitleForUndefinedLanguage = Utils.localizedString(this.props.config.locale, stringConstants.undefinedLanguageTitle, this.props.config.localizableStrings);
```

Comment:
I'm not sure we have to remove all of this, let's ask the person who introduced this change.

Reply:
Resolved.